recipes/shellcheck/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "shellcheck" %}
+{% set version = "0.5.0" %}
+{% set sha256 = "348a3f7892c1f28a44f188c00ac82f1b3bf899d9f81d14ddb0e306db26c937bb" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/koalaman/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - toolchain
```

Comment:
Please update to the use the new `compiler` syntax.

ref: https://conda-forge.org/docs/meta.html#compilers